builders/build/eth-api/libraries/viem.md

```diff
@@ -0,0 +1,416 @@
+---
+title: How to use viem Ethereum Library
+description: Check out this tutorial to learn how to use the viem TypeScript interface for Ethereum to send transactions and deploy Solidity smart contracts to Moonbeam.
+---
+
+# viem TypeScript Ethereum Library
+
+## Introduction {: #introduction }
+
+[viem](https://viem.sh/){target=_blank} is a modular TypeScript library that allows developers to interact with abstractions over the JSON-RPC API, making it easy to interact with Ethereum nodes. Since Moonbeam has an Ethereum-like API available that is fully compatible with Ethereum-style JSON RPC invocations, developers can leverage this compatibility to interact with Moonbeam nodes. For more information on viem, check out their [documentation site](https://viem.sh/docs/getting-started.html){target=_blank}.
```

Comment:
JSON-RPC has a hyphen in one spot and doesn't have one in another.

Reply:
Good catch! I actually just fixed this in a separate PR: https://github.com/moonbeam-foundation/moonbeam-docs/pull/760